nix/hydra/project.nix

```diff
@@ -60,6 +60,12 @@ let
           export GIT_REVISION=${gitRev}
         '';
       }
+      {
+        packages.hydra-tui.preBuild = ''
+          echo ======= PATCHING --version to ${gitRev} =======
+          export GIT_REVISION=${gitRev}
+        '';
```

Comment:
Could: re-use the string

Something like:

```nix
        let patchGitRevision = '' ... '';
        in {
          packages.hydra-node.preBuild = patchGitRevision;
          packages.hydra-tui.preBuild = patchGitRevision;
        }
```

Reply:
@ch1bo would you be so kind to propose a commit for that because I don't have the nix skills, it appears, to introduce constants in this context and make nix happy about it :(

Reply:
Done, Thank you for the help.